pkg/reader/ccip.go

```diff
@@ -565,10 +561,25 @@ func (r *ccipChainReader) bindReaderContract(
 	return nil
 }
 
-// newSync goes through the input contracts and binds them to the contract reader.
-//
-//nolint:unused // it will be used soon.
+func (r *ccipChainReader) Sync(ctx context.Context, _ ContractAddresses) error {
+	contracts, err := r.DiscoverContracts(ctx, r.destChain)
```

Comment:
I don't think `DiscoverContracts` will work if the offramp is not bound first, that's probably causing the int test to fail

Reply:
aaaah. ty!